examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -80,14 +80,14 @@
   {
```

Comment:
<p>I think we need an INPUT_DATA_DIR and OUTPUT_DATA_DIR.</p><p> </p><p>We need to store the NVTabular output to disk and do not keep the data into memory. This works well for small dataset, but this doesn't scale to only medium size dataset. We should design examples for at least medium size datasets.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>added the <code>OUTPUT_DATA_DIR</code></p>